core/controllers/subtopic_viewer.py

```diff
@@ -59,6 +59,26 @@ class SubtopicPageDataHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'GET': {
+            'topic_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Good point, I suppose it can not. 

But will it then need some sort of validator similar to https://github.com/oppia/oppia/blob/develop/core/controllers/topic_editor.py#L648 
???
So it would end up being like this?? 
```python
'topic_name': {
            'schema': {
                'type': 'basestring',
                'validators': [{
                    'id': 'has_length_at_most',
                    'max_value': constants.MAX_CHARS_IN_TOPIC_NAME
                }]
            }
        }
```

Or should I just remove the default??